src/components/ConferenceNewPage/ConferenceNewPage.tsx

```diff
@@ -85,18 +85,32 @@ const ConferenceNewPage: React.FC = () => {
   }
 
   const validateForm = (conference: Conference) => {
-    const { topic, startDate, endDate, city, country, name, url } = conference
+    const { topic, startDate, endDate, city, country, name, url, cfpUrl, twitter } = conference
+    const onlineRegex = /online|remote|everywhere|world|web|global|virtual|www|http/i
+    const isNotOnline = locationType !== 'online'
+    const httpRegex = /^http(s?):\/\//
+    const httpNoQuestionmarkRegex = /\?/
+    const urlShortener = /(\/bit\.ly)|(\/t\.co)/
+    const twitterRegex = /@(\w){1,15}$/
+
+    function isUrlValid(url:string) {
```

Comment:
You can move all these declaration (except `isNotOnline`) on line 68, before the component declaration. You can also make them `CONSTANTS`.

Reply:
done